airbyte-integrations/connectors/source-orb/source_orb/schemas/credits_ledger_entries.json

```diff
@@ -33,7 +33,7 @@
       "type": "string"
     },
     "credit_block_per_unit_cost_basis": {
-      "type": ["null", "string"]
+      "type": ["null", "number"]
```

Comment:
this will be a breaking change. can you bump the connector version with to 2.0.0 and write a migration guide? let me know if this is the first time and I'll share a template

Reply:
Yes please, it's my first time with the migration guide. Please share a template @girarda